integration-test-common/src/test/java/com/datastrato/gravitino/integration/test/util/ITUtils.java

```diff
@@ -85,6 +86,7 @@ public static void assertionsTableInfo(
       List<Column> columns,
       Map<String, String> properties,
       Index[] indexes,
+      Transform[] partitioning,
```

Comment:
we can add a new method to avoid some unnecessary changes of caller

Reply:
Since we already set partitioning when loading tables, I think it's a necessary change in `assertionsTableInfo` to check it for all of the existing and future tests.